test/acceptance/tests/basic/terraform/basic-install/main.tf

```diff
@@ -266,6 +268,82 @@ module "test_server" {
   consul_client_token_secret_arn = var.secure ? module.acl_controller[0].client_token_secret_arn : ""
   acl_secret_name_prefix         = var.suffix
   consul_ecs_image               = var.consul_ecs_image
+
+  // Test passing both a role resource and role data source objects to make sure both
+  // have the necessary fields ("arn" and "id").
+  task_role                     = aws_iam_role.task
+  execution_role                = data.aws_iam_role.execution
+  additional_task_role_policies = [aws_iam_policy.exec.arn]
```

Comment:
why are you adding the exec policies to the task role?

Reply:
Oh, this is bad naming. That's the policy to allow `aws execute-command`, which goes in the task role: https://github.com/hashicorp/terraform-aws-consul-ecs/blob/dd37985b6cb6608018772cd85f265a2585a1335c/modules/mesh-task/main.tf#L50-L87